### PR TITLE
fix(CampaignExport) #MLYSTOR-679: export can change name via config

### PR DESCRIPTION
**Pull request by Rseuret**
## Describe your changes
take config name
## Issue ticket number and link
https://jira.support-ent.fr/browse/MLYSTOR-679

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/lystore/export/campaign/CampaignExport.java [hunk 1]
```diff
@@ -23,35 +24,35 @@
...
-        this.supplierService = new DefaultSupplierService(Lystore.lystoreSchema, "supplier");
         this.id = id;
         this.ids = ids;
+        this.regionTypeName = regionTypeName;
         log.info("id CAMPAGNE "+ ids);
     }
 
 
     public void exportOrders(Handler<Either<String, Buffer>> handler) {
+        log.info(regionTypeName);
```
Comment (sjollois) [inline src/main/java/fr/openent/lystore/export/campaign/CampaignExport.java:46]: c'est normal de laisser un lof info d'une variable ?

**Review by sjollois — APPROVED** [review]

File: src/main/java/fr/openent/lystore/export/campaign/CampaignExport.java [hunk 2]
```diff
@@ -23,21 +24,20 @@
 import java.util.List;
 import java.util.Map;
 
+import static fr.openent.lystore.constants.ParametersConstants.REGION_TYPE_NAME;
```
Comment (JinDouang) [inline src/main/java/fr/openent/lystore/export/campaign/CampaignExport.java:27]: Tu n'as pas l'air de l'utiliser cet import

File: src/main/java/fr/openent/lystore/export/campaign/CampaignExport.java [hunk 3]
```diff
@@ -10,6 +10,7 @@
 import fr.openent.lystore.service.SupplierService;
 import fr.openent.lystore.service.impl.DefaultSupplierService;
 import fr.wseduc.webutils.Either;
+import fr.wseduc.webutils.I18n;
```
Comment (JinDouang) [inline src/main/java/fr/openent/lystore/export/campaign/CampaignExport.java:13]: Tu n'as pas l'air de l'utiliser cet import

File: src/main/java/fr/openent/lystore/export/campaign/CampaignExport.java [hunk 4]
```diff
@@ -23,21 +24,20 @@
...
 public class CampaignExport extends ExportObject {
-    private JsonObject params;
-    private ExportService exportService;
-    private Logger log = LoggerFactory.getLogger(fr.openent.lystore.export.validOrders.ValidOrders.class);
-    private String idFile;
-    private String url;
-    private Integer id;
-    private List<Integer> ids;
-    private SupplierService supplierService;
+    private final Logger log = LoggerFactory.getLogger(fr.openent.lystore.export.validOrders.ValidOrders.class);
```
Comment (JinDouang) [inline src/main/java/fr/openent/lystore/export/campaign/CampaignExport.java:30]: Certainement une indentation mais si tu peux le corriger la signature de la classe de ton logger :

  ```suggestion
    private final Logger log = LoggerFactory.getLogger(CampaignExport.class);
```

File: src/main/java/fr/openent/lystore/export/helpers/ExportHelper.java [hunk 5]
```diff
@@ -25,6 +27,8 @@
 import java.util.Calendar;
 import java.util.List;
 
+import static fr.openent.lystore.constants.ParametersConstants.REGION_TYPE_NAME;
```
Comment (JinDouang) [inline src/main/java/fr/openent/lystore/export/helpers/ExportHelper.java:30]: Tu ne l'utilises pas

**Review by JinDouang — CHANGES_REQUESTED** [review]

**Pull request merged**